tests/unit/mixins/fastboot-compat-test.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import FastbootCompatMixin from 'ember-user-activity/mixins/fastboot-compat';
+import { module, test } from 'qunit';
+
+module('Unit | Mixin | fastboot compat');
+
+// Replace this with your real tests.
```

Comment:
The fastboot service is only available when it is injected by the fastboot server, so `this.get('this.get('_isFastBoot')` will be `undefined`, ( the computed property `_fastboot` will return `undefined`) in the browser context.  To verify the existence of the fastboot service a test would need to be written that gets run as a result of being contained within a fastboot server instance, like using https://github.com/kaliber5/ember-fastboot-addon-tests .